### DISCUSSION TITLE
server error

### DISCUSSION THREAD

**Discussion by 61233** [DTB-consortium/Digital_twin_brain-open#1] [Q&A]
在昆山集群上运行server时出现以下类型的报错：
<img width="861" alt="e12cfb3c83a42eccdb17c401db3146b" src="https://github.com/DTB-consortium/Digital_twin_brain-open/assets/113967665/9edd50b3-5e67-4222-9b60-d9490616d89a">

**Comment by jimingda** [comment]
Hi @61233 

看起来是`j15r3n17`节点的问题，在`slurm`的 `--exclude` 选项中添加该节点试试
`sbatch --exclude  node1 node2 j15r3n17`
